src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarStatusScreenListener.java

```diff
@@ -187,38 +186,35 @@ public void onTownStatusScreen(TownStatusScreenEvent event) {
 							out.add(Translation.of("status_town_siege_status_warchest", warChest));
 						}
 
-						if(battleIsActive(siege)) {

```

Comment:
- The entire below section is just moved back one tab (no other changes)
- This is due to the removal of the IF statement